qt/branding.cpp

```diff
@@ -0,0 +1,102 @@
+// SPDX-FileCopyrightText: 2024 Carl Schwan <carl@carlschwan.eu>
+// SPDX-License-Identifier: LGPL-2.1-or-later
```

Comment:
Can you use the full copyright please, like in all other files? I do like the SPDX strings, but I think it will be easier to convert everything all at once at a future point, rather than having one outlier file.